cli/flags.rs

```diff
@@ -3665,7 +3712,7 @@ mod tests {
     assert_eq!(
       r.unwrap(),
       Flags {
-        subcommand: DenoSubcommand::Test {
+        subcommand: DenoSubcommand::Test(TestFlags {
```

Comment:
Tests could be simpler if we used defaults.

Reply:
Good point, but I think this could be done in a follow up

Reply:
SGTM.